packages/ruby-foreman/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+#
+# This PKGBUILD was extracted from AUR.
+# Creator: Charlie Wolf <charlie@wolf.is>
+# Changes by: petruknisme
+
+
+_gemname=foreman
+pkgname=ruby-${_gemname}
+pkgver=0.87.1
+pkgrel=1
+pkgdesc="Process manager for Procfile based applications"
+arch=('any')
+depends=('ruby' 'ruby-thor')
+url="https://rubygems.org/gems/${_gemname}"
+noextract=($_gemname-$pkgver.gem)
+license=('MIT')
+source=(
+	"https://rubygems.org/downloads/${_gemname}-${pkgver}.gem"
+)
+sha512sums=(
+	'cc511cca4afe90ab784af60bf07420d2ae36bbd3e58689e684be66dc47c24df231fd4b793d53698616a8c6ec6a02b9f1dc0210111613edaf18b636da3c3767cc'
+)
+
+package() {
+	local _gemdir="$(ruby -e'puts Gem.default_dir')"
+	gem install --ignore-dependencies --no-user-install -i "$pkgdir/$_gemdir" -n "$pkgdir/usr/bin" $_gemname-$pkgver.gem
+	rm "$pkgdir/$_gemdir/cache/$_gemname-$pkgver.gem"
+}
```

Comment:
Can you check is there some tests and specs to remove, and also copy the LICENSE since it's a MIT license.

As an example https://github.com/BlackArch/blackarch/blob/master/packages/ruby-docopt/PKGBUILD

Reply:
I see. Wait i will recheck

Reply:
There is no test and examples.
```
Successfully installed foreman-0.87.1
1 gem installed
rm: cannot remove '/build/ruby-foreman/pkg/ruby-foreman//usr/lib/ruby/gems/2.7.0/gems/foreman-0.87.1/test': No such file or directory
rm: cannot remove '/build/ruby-foreman/pkg/ruby-foreman//usr/lib/ruby/gems/2.7.0/gems/foreman-0.87.1/examples': No such file or directory
```